core/templates/pages/exploration-editor-page/settings-tab/settings-tab.component.ts

```diff
@@ -335,6 +337,10 @@ angular.module('oppia').component('settingsTab', {
         AlertsService.clearWarnings();
 
         var moderatorEmailDraftUrl = '/moderatorhandler/email_draft';
+        var onUnpublishingcallback = function() {
```

Comment:
Maybe name this function based on what it does? (Something around `fetchAndUpdatePermissions`.)